modules/ballerina-builtin/src/main/ballerina/ballerina/security/crypto/natives.bal

```diff
@@ -0,0 +1,14 @@
+package ballerina.security.crypto;
+
+@Description {value:"Returns a hash of a given string using the specified algorithm (supported algorithms: SHA1, SHA256, MD5)."}
+@Param {value:"baseString: The string to be hashed"}
+@Param {value:"algorithm: The hashing algorithm to be used"}
+@Return {value:"The hashed string"}
+public native function getHash (string baseString, string algorithm) (string);
```

Comment:
Shall we make this algorithm  an enum in crypto package? Having it as a string is more prone to typos and user has to refer the docs to find the supported algorithms